specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,710 @@
+                "type": "string"
+              }
+            },
+            {
+              "name": "reason",
+              "in": "query",
+              "description": "Close reason.",
+              "schema": {
+                "type": "string"
+              }
+            }
+          ],
+          "responses": {
+            "202": {
+              "description": "Accepted"
+            }
+          }
+        }
+      },
+      "/api/v1/hubs/{hub}/groups/{group}": {
+        "post": {
+          "description": "Broadcast a message to all clients within the target group.",
+          "tags": [],
+          "parameters": [
+            {
+              "name": "hub",
+              "in": "path",
+              "required": true,
+              "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+              "schema": {
+                "type": "string"
+              }
+            },
+            {
+              "name": "group",
+              "in": "path",
+              "required": true,
+              "description": "Target group name, which length should be greater than 0 and less than 1025.",
+              "schema": {
+                "type": "string"
+              }
+            }
+          ],
+          "requestBody": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/PayloadMessage"
+                }
+              }
+            }
+          },
+          "responses": {
+            "202": {
+              "description": "Ok"
+            }
+          }
+        }
+      },
+      "/api/v1/hubs/{hub}/groups/{group}/users/{user}": {
+        "get": {
+          "description": "Check whether a user exists in the target group.",
+          "tags": [],
+          "parameters": [
+            {
+              "name": "hub",
+              "in": "path",
+              "required": true,
+              "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+              "schema": {
+                "type": "string"
+              }
+            },
+            {
+              "name": "group",
+              "in": "path",
+              "required": true,
+              "description": "Target group name, which length should be greater than 0 and less than 1025.",
+              "schema": {
+                "type": "string"
+              }
+            },
+            {
+              "name": "user",
+              "in": "path",
+              "required": true,
+              "description": "Target user Id.",
+              "schema": {
+                "type": "string"
+              }
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "Ok"
+            },
+            "404": {
+              "description": "NotFound"
```

Comment:
See previous comment on x-ms-error-response

Reply:
Hey @Imazuel, thank you for taking the time to look into the PR and provide valuable feedbacks!

For this `404` one, this API is to check existence, and we leverage `404` to indicate that the resource does not exist. So the semantics of `404` is different than 5xx errors. Is `x-ms-error-response` also applicable in this case?

Reply:
I saw a GET verb, so I assumed you wanted to return a User or nothing. I see you indeed didn't have a schema, so I guess it's fair.
If your API supports it, checks are usually models using HEAD, so save some body on the wire (just saying).

Reply:
We add both `HEAD` and `GET`, for `GET`, the difference is the error response contains detailed error when `400 Bad Request`.  Shall we add `x-ms-error-response` to `4xx` responses?